cluster-operator/src/test/java/io/strimzi/operator/cluster/operator/assembly/KafkaConnectBuildAssemblyOperatorKubeTest.java

```diff
@@ -567,8 +567,8 @@ public void testContinueWithPreviousBuildOnKube(VertxTestContext context) {
                 .endSpec()
                 .build();
 
-        KafkaConnectCluster oldConnect = KafkaConnectCluster.fromCrd(oldKc, VERSIONS);
-        KafkaConnectBuild oldBuild = KafkaConnectBuild.fromCrd(oldKc, VERSIONS);
+        KafkaConnectCluster oldConnect = KafkaConnectCluster.fromCrd(Reconciliation.DUMMY_RECONCILIATION, oldKc, VERSIONS);
+        KafkaConnectBuild oldBuild = KafkaConnectBuild.fromCrd(new Reconciliation("test", oldKc.getKind(), oldKc.getMetadata().getNamespace(), oldKc.getMetadata().getName()), oldKc, VERSIONS);
```

Comment:
Any reason this does not use DUMMY_RECONCILIATION? Same also for the other cases in this file.